packages/fast-components-react-base/package.json

```diff
@@ -115,6 +115,7 @@
   "dependencies": {
     "@microsoft/fast-components-class-name-contracts-base": "^4.6.0",
     "@microsoft/fast-components-foundation-react": "^3.1.3",
+    "@microsoft/fast-tooling": "^0.0.0",
```

Comment:
Will this need to get published to `npm` by the OS team prior to `lerna publish`?

Reply:
Ah nm - this is in a releases branch. When are you intending to merge this to master?

Reply:
When the major release of tooling v2 goes in.

I've actually just adjusted this and created v2 of the schemas to ensure there is no breaking change in any dependencies that import these files.